### ISSUE TITLE
Feedback to user on download

### ISSUE THREAD

**Issue by HIPPhilipp** [dCache/dcache-view#269]
I am not really sure what happens, but when starting the download of a larger file, no feedback is given to the user. 

It looks like the file is downloaded first and the dialogue if and where you want to save this file locally appears after it is downloaded. In consequence one is tempted to start a download multiple times.

**Comment by ZNikke** [comment]
Yeah, it indeed looks like the file is fetch():ed first and then passed on to the browser to process only when the fetch() completes. I guess it makes sense on small files that you potentially want to show directly in dCacheView, but it's not something I would expect when I explicitly select to download it...

[dv.js#L635](https://github.com/dCache/dcache-view/blob/master/src/scripts/dv.js#L635) and [download-task.js](https://github.com/dCache/dcache-view/blob/master/src/scripts/tasks/download-task.js) seems to be the relevant code.

I suspect that the correct solution would be to rip all the download stuff out and replace it with something that lets the browser do the work, which would include first opening the expected save-dialog. Or rename the current download-code to a View action and limit it to handle things that could be displayed inline in dCacheView.

Since https://github.com/dCache/dcache-view/issues/268 is also related to download-handling, it would likely make sense to look at both issues at the same time.

**Comment by ZNikke** [comment]
Also, the current download code seems prone to not report errors.

Using Firefox on Linux, I tried downloading a file significantly larger than my /tmp file system since that's where the file seems to land for me before I get the save-dialog. Anyhow, I could see that a temporary file showed up, grew in size until fs was full, and then vanished. No error notification/dialog or anything.
